doc/developer-guides/hld/hv-dev-passthrough.rst

```diff
@@ -60,14 +63,39 @@ passthrough, as detailed here:
 
    Passthrough Device Status
 
-DMA Remapping
-*************
+Owner of Passthrough Devices
+****************************
+
+ACRN hypervisor will do PCI enumeration to discover the PCI devices on the platform.
+According to the hypervisor/VM configurations, the owner of these PCI devices can
+the following 4 cases:
+
+- **Hypervisor**: hypervisor uses UART device as the console in debug version for
```

Comment:
Is this the case for the UART only, or are you using the UART as an example of such device that is assigned and visible only to the hypervisor?

Reply:
currently is the only one, I will add a description.